src/request.ts

```diff
@@ -37,7 +37,7 @@ const _tryParseJSON = (res: Response): Promise<any> => {
  */
 export const _fetchJSON = (
     url: string,
-    {headers, method, mode, ...options}: RequestInit = {}
+    {headers = {}, method = 'GET', mode, ...options}: RequestInit = {}
```

Comment:
Another way to accomplish this would be to update the if below to match how we do it internally. 

`if (method && method !== 'GET') {`

Reply:
Probably should go with the other way. Otherwise a lot of internal code would break.

Reply:
Internally, we also default it, but we do it later in this function. I would imagine the internal function was written pre-es6 defaults